ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -310,10 +310,37 @@
 {%     if port_channel_interfaces_isis | length > 0 %}
  *Inherited from Port-Channel Interface
 {%     endif %}
+{%     set err_cor_enc_intfs = [] %}
+{%     for ethernet_interface in ethernet_interfaces %}
+{%         if ethernet_interfaces[ethernet_interface].error_correction_encoding is arista.avd.defined %}
+{%             do err_cor_enc_intfs.append(ethernet_interface) %}
+{%         endif %}
+{%     endfor %}
+{%     if err_cor_enc_intfs | length > 0 %}
+
+#### Error Correction Encoding Interfaces
+
+| Interface | Enabled |
+| --------- | ------- |
+{%         for ethernet_interface in err_cor_enc_intfs | arista.avd.natural_sort %}
+{%             if ethernet_interfaces[ethernet_interface].error_correction_encoding.enabled is arista.avd.defined(false) %}
+{%                 set enabled = ['Disabled'] %}
+{%             else %}
+{%                 set enabled = [] %}
+{%                 if ethernet_interfaces[ethernet_interface].error_correction_encoding.fire_code is arista.avd.defined(true) %}
+{%                     do enabled.append('fire-code') %}
+{%                 endif %}
+{%                 if ethernet_interfaces[ethernet_interface].error_correction_encoding.reed_solomon is arista.avd.defined(true) %}
+{%                     do enabled.append('reed-solomon') %}
+{%                 endif %}
+{%             endif %}
+| {{ ethernet_interface }} | {{ enabled | join('<br>') }} |
```

Comment:
please join on `, `.

Reply:
I was about to create a new PR for this issue and was actually wondering...
Is there a need for the enabled option?

the way I have currently implemented is the following:
error_correction_encoding:
  encodings:
  - fire_code
  - reed_solomon

Since the default values on EOS are set to False for both options, and since we can enable either or both. I implemented it this way. 

If you aggree with my method, then I will either commit to this PR, or make my own PR.

Reply:
We need the `enabled` flag to be able to configure `no error-correction encoding` - hence to disable error correction completely. Please check the issue for the discussion.

Reply:
> We need the enabled flag to be able to configure no error-correction encoding - hence to disable error correction completely. Please check the issue for the discussion.

Actually agreed, after checking the default value on EOS.

Reply:
I thought it would be good looking if we join with ```<br>``` than ```,``` because entries are added to different lines which makes it easy to differentiate the items.